### PR TITLE
create To-do-list

### PR DESCRIPTION
**Pull request by Christelle-12**
Hello , In this milestone I created :

- Created an index.html file and write my HTML markup here. 
- Created an index.js file and set an array of some simple to-do tasks (array of objects). Each task object contains three keys:
description [string].
completed [bool].
index: [number].
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- rendered the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Created a style.css and set rules for the To-Do List.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      </div>
+      <div class="task-item__description ${task.completed ? 'completed' : ''}">
+        ${task.description}
+      </div>
+      <div class="task-item__delete">
+        <button class="delete-button" data-index="${task.index}">
+          <i class="fas fa-trash-alt"></i>
+        </button>
+      </div>
+    </div>
```
Comment (tresorsawasawa) [inline src/index.js:35]: - [ ] Once you have fixed all the above changes and make sure that your application is running, you should ensure that created CSS is  used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) as mentioned in the project requirements. Right now it is difficult to check as you didn't correctly setup your Webpack dev server

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Christelle-12 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+<body>
+  <header>
+      <h1>Today's To Do</h1>
+      <i class="fa-solid fa-rotate"></i>
+  </header>
+  <ul id="task-list"></ul>
+  <button id="clear-btn">Clear Completed Tasks</button>
+  <script src="/dist/main.js"></script>
+</body>
```
Comment (GedeonTS) [inline src/index.html:20]: - As requested by the previous reviewer [here](https://github.com/Christelle-12/To-do-list/pull/1#discussion_r1160560923) kindly style your project as requested in the project requirements [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Christelle-12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Descriptive PR message :+1: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Christelle-12 ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good job styling your work :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**